proxy/rpc/server_abc.py

```diff
@@ -354,6 +354,7 @@ async def _init_genesis_block(self) -> None:
             block_time=block_time,
             parent_slot=0,
             parent_block_hash=parent_hash,
+            priority_fee_percentiles=[0] * 11,
```

Comment:
Please rename to:
`priority_fee_percentile_list`

Reply:
Please move the magic constant to NeonBlockHdr

Reply:
done